sale_restricted_qty/models/sale.py

```diff
@@ -48,18 +48,16 @@ class SaleOrderLine(models.Model):
         "product_uom_qty", "sale_min_qty", "sale_max_qty", "sale_multiple_qty"
     )
     def check_constraint_restricted_qty(self):
-
         msg = ""
         invaild_min_lines = []
         line_to_test = self.filtered(
             lambda sl: not sl.product_id.force_sale_min_qty and sl.is_qty_less_min_qty
         )
         for line in line_to_test:
             invaild_min_lines.append(
-                _('Product "%s": Min Quantity %s.')
-                % (line.product_id.name, line.sale_min_qty)
+                _("Product %(name)s: Min Quantity %(qty)s.")
+                % {"name": line.product_id.name, "qty": line.sale_min_qty}
```

Comment:
You can use parameters of translation function to do that:
` _("Product %(name)s: Min Quantity %(qty)s.", name=line.product_id.name, qty=line.sale_min_qty)`

Reply:
Same for others